pkg/ddl/executor.go

```diff
@@ -10105,20 +10111,29 @@ func (e *executor) notifyNewJobByEtcd(etcdPath string, jobID int64, jobType stri
 }
 
 func (e *executor) deliverJobTask(task *JobWrapper) {
-	if task.LocalMode {
-		e.limitJobChV2 <- task
-	} else {
-		e.limitJobCh <- task
-	}
+	// TODO this might block forever, as the consumer part considers context cancel.
+	e.limitJobCh <- task
 }
 
 // addBatchDDLJobsV1 gets global job IDs and puts the DDL jobs in the DDL queue.
 func (d *ddl) addBatchDDLJobsV1(jobWs []*JobWrapper) {
```

Comment:
maybe we can revert the name to `addBatchDDLJobs` in later pr.

Reply:
lgtm